### DISCUSSION TITLE
Transmeta Crusoe

### DISCUSSION THREAD

**Discussion by AITUS95** [86Box/86Box#2895] [Ideas]
I found in the archive some documents regarding the transmeta crusoe cpu, it would be nice if it were possible to add some variant of this cpu, I don't know if it could help the emulation in some way also because it is a cpu that emulates x86 instructions, but it's still part of the story, I'm attaching the documentation:

[TM5400_ProductBrief_5-23-00.pdf](https://github.com/86Box/86Box/files/10094385/TM5400_ProductBrief_5-23-00.pdf)
[TM3200_ProductBrief_5-20-00.pdf](https://github.com/86Box/86Box/files/10094386/TM3200_ProductBrief_5-20-00.pdf)
[CrusoeBenchmarkReport_5-20-00.pdf](https://github.com/86Box/86Box/files/10094387/CrusoeBenchmarkReport_5-20-00.pdf)
[BenchmarkWhitePaper_2-3-00f__2_.pdf](https://github.com/86Box/86Box/files/10094388/BenchmarkWhitePaper_2-3-00f__2_.pdf)
[crusoetechwp.pdf](https://github.com/86Box/86Box/files/10094389/crusoetechwp.pdf)
[crusoe_tm5700-tm5900_processor.pdf](https://github.com/86Box/86Box/files/10094390/crusoe_tm5700-tm5900_processor.pdf)
[TM5400_DataSheet_1-18-00 (1).pdf](https://github.com/86Box/86Box/files/10094391/TM5400_DataSheet_1-18-00.1.pdf)

Source:
https://web.archive.org/web/20000815064600/http://www.transmeta.com/crusoe/

**Comment by OBattler** [comment]
Converting this to a discussion.
